test/api/application/fixtures.go

```diff
@@ -0,0 +1,67 @@
+package application
+
+import (
+	"github.com/konveyor/tackle2-hub/api"
+	"github.com/konveyor/tackle2-hub/test/api/client"
+)
+
+var (
+	// Setup Hub API client
+	Client = client.Client
```

Comment:
@jortel Import aliases were removed, assertions moved to assert package. Removal of the Client variable setup leading to ```client.Client.Post(...)``` on many lines in tests did look too strange to me, so I left it as it was.

If we are strongly opinionated about removing Client variable when other non-client methods are called via their package names, I'd  consider some kind of rename client.Client to client.Api/API to be ```client.API.Post(...)``` that looks better to me.